x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -55,57 +69,182 @@ export const OutlierExploration: FC<ExplorationProps> = React.memo(({ jobId }) =
     jobConfig !== undefined ? getFeatureCount(jobConfig.dest.results_field, tableItems) : 1
   );
 
+  const [expandedRowItem, setExpandedRowItem] = useState<DataFrameAnalyticsListRow | undefined>(
+    undefined
+  );
+
+  const fetchStats = async () => {
+    const analyticsConfigs = await ml.dataFrameAnalytics.getDataFrameAnalytics(jobId);
+    const analyticsStats = await ml.dataFrameAnalytics.getDataFrameAnalyticsStats(jobId);
+
+    const config = analyticsConfigs.data_frame_analytics[0];
+    const stats = isGetDataFrameAnalyticsStatsResponseOk(analyticsStats)
+      ? analyticsStats.data_frame_analytics[0]
+      : undefined;
+
+    if (stats === undefined) {
+      return;
+    }
+
+    const newExpandedRowItem: DataFrameAnalyticsListRow = {
+      checkpointing: {},
+      config,
+      id: config.id,
+      job_type: Object.keys(config.analysis)[0] as DataFrameAnalysisConfigType,
```

Comment:
Suggestion to use `getAnalysisType` here

Reply:
Fixed in 819cf7d.